include/readdy/kernel/singlecpu/model/SCPUNeighborList.h

```diff
@@ -70,18 +70,21 @@ class CellLinkedList {
     CellLinkedList(data_type &data, const readdy::model::Context &context)
             : _data(data), _context(context), _head{}, _list{}, _radius{0} {};
 
-    void setUp(scalar interactionDistance, cell_radius_type radius) {
-        if (!_isSetUp || _interactionDistance != interactionDistance || _radius != radius) {
-            if (interactionDistance < _context.get().calculateMaxCutoff()) {
-                throw std::logic_error(fmt::format(
-                        "The requested interaction distance {} for neighbor-list set-up was smaller than the largest cutoff {}",
-                        interactionDistance, _context.get().calculateMaxCutoff()));
+    void setUp(scalar cutoff, cell_radius_type radius) {
+        if (!_isSetUp || _cutoff != cutoff || _radius != radius) {
+            if (cutoff <= 0) {
+                throw std::logic_error("The cutoff distance for setting up a neighbor list must be > 0");
+            }
+            if (cutoff < _context.get().calculateMaxCutoff()) {
+                log::warn(fmt::format(
+                        "The requested cutoff {} for neighbor-list set-up was smaller than the largest cutoff {}",
+                        cutoff, _context.get().calculateMaxCutoff()));
             }
             _radius = radius;
-            _interactionDistance = interactionDistance;
-            if (_interactionDistance > 0) {
+            _cutoff = cutoff;
+            if (_cutoff > 0) {
```

Comment:
this check is now unnecessary since you throw when cutoff is <= 0 :)

Reply:
removed